testing/contract/examples/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# shellcheck disable=SC1091
+source "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"/setup.sh
+
+# shellcheck disable=SC2154
```

Comment:
We need to address this warning, there are workarounds proposed for this in https://www.shellcheck.net/wiki/SC2154.

Reply:
The listed suggestions did not help, but i found another solution, let me know what you think.